mix.exs

```diff
@@ -12,7 +12,6 @@ defmodule ImgProxy.MixProject do
       elixir: "~> 1.8",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
-      description: "imgproxy URL generator and helper functions",
```

Comment:
The [docs](https://hex.pm/docs/publish#adding-metadata-to-code-classinlinemixexscode) seem to indicate this is a key that should be at the top level (see the [reference example file](https://hex.pm/docs/publish#documentation))